docs/operate-and-deploy/installation/upgrading.md

```diff
@@ -42,6 +42,30 @@ upgrading a cluster involves leaving the old cluster running on the old version,
 cluster on the new version, porting across your database schema, and finally thinking about your data.
 Read on for details.
 
+
+## (1) In-Place upgrade
+
+#### Steps :-
+
+1. Download the [new ksqlDB version](https://docs.confluent.io/platform/current/installation/installing_cp/zip-tar.html#get-the-software) to upgrade to.
+2. Set it up with all required configs. In our case we mostly copied all configs from the current version(which is running) to the new version.
+3. Take Backup and then copy all UDF JAR to new version .(OPTIONAL STEP In case you don't have any UDF).
+4. Stop ksqlDB process of current version(5.5 in our case) and start newer version(6 in our case) with latest KsqlDB config file which you created in step2.
```

Comment:
```suggestion
4. Stop ksqlDB process of current version and start newer version with latest ksqlDB config file which you created in step 2.
```
Minor rewrite to remove parentheticals.

Reply:
@vcrfxia actually i missed all these comments, i have made all suggested minor changes.

Reply:
Let me know if something is missing.